ExerciseTwo/exercise.html

```diff
@@ -1,38 +1,33 @@
-<!-- 
-    INSTRUCTIONS:
-    Fix the 17 lines with indentation issues below.
--->
 <!DOCTYPE html>
-<head>
-            <title>Some title</title>
-    </head>
-    <body>
+<html>
+  <head>
+    <title>Some title</title>
+  </head>
+  <body>
     <div>
-        <p>Test One</p>
-                <div>
-                <p>Test Two</p>
-                    <div>
-                    <p>Test Three</p>
-                        <div>
-                            <p>Test Four</p>
-                        <div>
-                        <p>Test Five</p>
-                    </div>
-                    </div>
-                    <section>
-                        <span>
-                            <ul>
-                                        <li>
-                                        <div>Something</div>
-                                </li>
-                                <li>
-                                    <div>Something else </div>
-                                </li>
-                                </ul>
-                            </span>
-                    </section>
-                    </div>
-        </div>
-        </div>
-        </body>
+      <p>Test One</p>
+    </div>
+    <div>
+      <p>Test Two</p>
+    </div>
+    <div>
+      <p>Test Three</p>
+    </div>
+    <div>
+      <p>Test Four</p>
+    </div>
+    <div>
+      <p>Test Five</p>
+    </div>
+    <div>
+      <section>
+        <ul>
+          <li>Something</li>
+            <span></span>
+          <li>Something else</li>
+            <span></span>
```

Comment:
You have an extra space before this `<span>`

Reply:
This `<span>` also does not belong within a `<ul>` tag.
Only `<li>` (list item elements) can be used within `<ul>`